experimental/traffic-portal/src/app/api/topology.service.spec.ts

```diff
@@ -0,0 +1,70 @@
+/**
```

Comment:
The coverage provided by this spec is 

| Statements | Branches | Functions | Lines   |
|------------------|---------------|----------------|-----------|
|        42.55% |    27.27%  |        37.5% |  41.3% |

but it _should_ be


| Statements | Branches | Functions | Lines   |
|------------------|---------------|----------------|-----------|
|            100% |       100%  |         100% |   100% |

like (most of) the other API services

Reply:
`topology.service` coverage increased to 100% in 0e5b61c515